src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -309,12 +309,19 @@ export function PostDialogUI(props: PostDialogUIProps) {
 export interface PostDialogProps extends Omit<Partial<PostDialogUIProps>, 'open'> {
     open?: [boolean, (next: boolean) => void]
     reason?: 'timeline' | 'popup'
+    commentId?: string
```

Comment:
We need to hook each postbox and its corresponding popup editor box, otherwise it will popup multiple editor boxes.